spec/integration/sidekiq_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+require 'timecop'
+
+Sidekiq::Testing.fake!
+
+class MySidekiqWorker
+  include Sidekiq::Worker
+
+  def perform(_name); end
+end
+
+describe 'Sidekiq client logger' do
+  let(:target) { StringIO.new }
+
+  before do
+    Loga.reset
+
+    Loga.configure(
+      service_name: 'hello_world_app',
+      service_version: '1.0',
+      device: target,
+      format: :gelf,
+    )
+
+    Sidekiq::Logging.logger = Loga.logger
+  end
+
+  let(:json_line) do
+    target.rewind
+    JSON.parse(target.read)
+  end
+
+  it 'has the proper job logger' do
+    job_logger = Loga::Sidekiq::JobLogger
+
+    expect(Sidekiq.options[:job_logger]).to eq job_logger
+  end
+
+  it 'logs the job enqueue' do
+    MySidekiqWorker.perform_async('Bob')
+
+    expected_body = {
+      '_service.name' => 'hello_world_app',
+      '_service.version' => '1.0',
+      'version' => '1.1',
+      'level' => 6,
+      '_tags' => '',
+    }
+
+    aggregate_failures do
+      expect(json_line).to include(expected_body)
+      expect(json_line['timestamp']).to be_a(Float)
+      expect(json_line['host']).to be_a(String)
+      expect(json_line['short_message']).to match(/Booting Sidekiq \d\.\d\.\d/)
```

Comment:
This is the init message. There is a message when a job gets executed, not when scheduled. TBO I have no idea how we can test this.. I'll check the gem internals shortly